common/ts/sonarqube/Analysis.ts

```diff
@@ -62,7 +63,7 @@ export default class Analysis {
       font-size: 12px;
       margin-left: 15px;`;
     return `<div style="padding-top: 8px;">
-      <span>Quality Gate</span>
+      <span>${this.projectName} Quality Gate</span>
```

Comment:
We don't want to add the project name if there is only one analysis to be displayed.